test/unit/pipeline/test_create_basecase_pipeline.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from test.utils import BaseTestCase, TestData
+import pytz
+
+import numpy as np
+import pandas as pd
+
+from datetime import datetime, timezone, timedelta
+
+from openstf.pipeline.create_basecase_forecast_sklearn import basecase_pipeline
+
+NOW = datetime.now(timezone.utc)
+
+
+class TestBaseCaseForecast(BaseTestCase):
+    def test_basecase_pipeline_happy_flow(self):
+        # Test happy flow
+        PJ = TestData.get_prediction_job(pid=307)
+        forecast_input = TestData.load("reference_sets/307-test-data.csv")
+
+        # Shift example data to match current time interval as code expects data
+        # available relative to the current time.
+        utc_now = (
+            pd.Series(datetime.utcnow().replace(tzinfo=pytz.utc))
+            .min()
+            .round("15T")
+            .to_pydatetime()
+        )
+        most_recent_date = forecast_input.index.max().ceil("15T").to_pydatetime()
+        delta = utc_now - most_recent_date + timedelta(3)
```

Comment:
This works. Alternatively (see it as a suggestion for next yime) you could mock the 'datetime.utcnow()' function, so this returns a pre-defined datetime. Like that, you are certain the input of the function is always the same when the test is executed